Completion/Unix/Type/_remote_files

```diff
@@ -30,12 +30,12 @@
 # There should be coloring based on all the different ls -F classifiers.
 local expl rempat remfiles remdispf remdispd args cmd suf ret=1
 local -a args cmd_args
-local glob host
+local glob host dir dirprefix
 
 if zstyle -T ":completion:${curcontext}:files" remote-access; then
 
   # Parse options to _remote_files. Stops at the first "--".
-  zparseopts -D -E -a args / g:=glob h:=host
+  zparseopts -D -E -a args / g:=glob h:=host d:=dir
```

Comment:
Could you add the new flag to the documentation (lines 6 through 13)?

Reply:
@danielshahaf Thanks for your review, I've added the docs.

Reply:
This change looks fine to me, the one thing I wonder is whether the switch should be `-W` for consistency with `_files`. `-d` is a compadd option, albeit one that makes little sense here but they tend to be best avoided. I don't think it necessarily needs to replicate the `_files` support for an array with `-W` – that doesn't get much use anyway,

Reply:
@okapia Thanks for your help, I've updated the code to switch `-d` with `-W`.